### PR TITLE
Asset: added test cases for asset and VC over issuance

### PR DESCRIPTION
**Pull request by atreyasuraj**

### CODE REVIEW

File: pallets/asset/src/tests.rs [hunk 1]
```diff
@@ -138,29 +138,6 @@ fn asset_issue_should_succeed() {
 
 	let issue_entry_digest =
 		<Test as frame_system::Config>::Hashing::hash(&[&issue_entry.encode()[..]].concat()[..]);
-
-	new_test_ext().execute_with(|| {
```
Comment (vatsa287) [inline pallets/asset/src/tests.rs]: nit: Maybe this has been removed by mistake.

File: pallets/asset/src/tests.rs [hunk 2]
```diff
@@ -874,3 +852,185 @@ fn changing_status_of_vc_asset_with_same_status_should_fail() {
...
+				issue_entry_digest,
+				authorization_id
+			),
+			Error::<Test>::OverIssuanceLimit
+		);
+	});
+}
+
+#[test]
+fn asset_vc_status_change_should_not_succeed() {
```
Comment (vatsa287) [inline pallets/asset/src/tests.rs:936]: Please change the test-case name since this already exists.

File: pallets/asset/src/tests.rs [hunk 3]
```diff
@@ -874,3 +852,185 @@ fn changing_status_of_vc_asset_with_same_status_should_fail() {
...
+		);
+
+		assert_err!(
+			Asset::vc_status_change(
+				DoubleOrigin(author.clone(), creator.clone()).into(),
+				asset_id.clone(),
+				Some(instance_id.clone()),
+				AssetStatusOf::INACTIVE
+			),
+			Error::<Test>::AssetInstanceNotFound
```
Comment (vatsa287) [inline pallets/asset/src/tests.rs:1033]: nit: Last test may not be required.

**Review by vatsa287 — DISMISSED** [review]
LGTM
nit: Would be right if the test case name be related to over issuance, as there might multiple ways issue, status_change may fail.

**Comment by atreyasuraj** [comment]
> LGTM nit: Would be right if the test case name be related to over issuance, as there might multiple ways issue, status_change may fail.



> LGTM nit: Would be right if the test case name be related to over issuance, as there might multiple ways issue, status_change may fail.

Sure, fixed!

**Review by amarts — APPROVED** [review]

**Pull request merged**